test/qa/lib/meshnet.py

```diff
@@ -176,9 +176,46 @@ def revoke_all_invites_in_peer(ssh_client: ssh.Ssh):
 
 
 def send_meshnet_invite(email):
-    command = sh.nordvpn.meshnet.invite.send(email)
-    process = subprocess.Popen(str(command), shell=True, stdin=subprocess.PIPE, stdout=subprocess.PIPE, stderr=subprocess.PIPE, text=True)
+    try:
+        command = ["nordvpn", "meshnet", "invite", "send", email]
+        process = subprocess.Popen(command, stdout=subprocess.PIPE, stdin=subprocess.PIPE, stderr=subprocess.PIPE, text=True)
 
-    for _ in range(4):
-        process.stdin.write('\n')
-        process.stdin.flush()
+        for _ in range(4):
+            process.stdin.write('\n')
+            process.stdin.flush()
+
+        stdout, stderr = process.communicate()
```

Comment:
Maybe add a `timeout` argument to `communicate()` to prevent test hang up if something goes wrong?

    try:
        stdout, stderr = process.communicate(timeout=5)
    except subprocess.TimeoutExpired:
        process.kill()
        stdout, stderr = process.communicate()

Reply:
good point, added this, thanks!